tests/contrib/django/test_connection.py

```diff
@@ -30,7 +26,7 @@ def test_connection(self):
 
         span = spans[0]
         eq_(span.name, 'sqlite.query')
-        eq_(span.service, 'defaultdb')
+        eq_(span.service, 'sample-defaultdb')
```

Comment:
as far as I remember it may be not so easy, but it would be great having also one test with just `defaultdb` if `DEFAULT_DATABASE_PREFIX` is not set